fractal_server/app/routes/api/v2/task.py

```diff
@@ -66,6 +75,25 @@ async def get_list_task(
     if args_schema_non_parallel is False:
         for task in task_list:
             setattr(task, "args_schema_non_parallel", None)
+    if authors is not None:
```

Comment:
This does work, but let's get back to a single string so that we can include the query in sql